src/client/store/ui/assessmentFiles/hooks/index.ts

```diff
@@ -1,5 +1,68 @@
-import { useAppSelector } from 'client/store'
```

Comment:
https://github.com/openforis/fra-platform/pull/3215#discussion_r1390384346

Related to this comment, should we move this to

`src/client/store/data/...`

If so, what would the store look like?
Example?: `state.data[assessmentName][cycleName][countryIso]: Array<AssessmentFile>`
Should we separate the global files to under WO?